### PR TITLE
dev-python/doit update to 0.33.1

### PR DESCRIPTION
**Pull request by winterheart**
Update to 0.33.1, remove old 0.32.0-r1

### CODE REVIEW

File: dev-python/doit/doit-0.33.1.ebuild [hunk 1]
```diff
@@ -3,7 +3,7 @@
 
 EAPI=7
 
-PYTHON_COMPAT=( python3_{6,7} )
```
Comment (thesamesam) [inline dev-python/doit/doit-0.33.1.ebuild:6]: Any chancce of 9?

Comment (winterheart) [reply dev-python/doit/doit-0.33.1.ebuild:6]: Need python/configclass[python_targets_python3_9] for that:

```
   dev-python/doit-py/doit-py-0.5.0.ebuild: BDEPEND: amd64(default/linux/amd64/17.0)
[     'dev-python/configclass[python_targets_python3_6(-)?,python_targets_python3_7(-)?,python_targets_python3_8(-)?,python_targets_python3_9(-)?,-python_single_target_python3_6(-),-python_single_target_python3_7(-),-python_single_target_python3_8(-),-python_single_target_python3_9(-)]']
```

Comment (thesamesam) [reply dev-python/doit/doit-0.33.1.ebuild:6]: Fancy adding it, if it works & tests pass there too?

Comment (thesamesam) [reply dev-python/doit/doit-0.33.1.ebuild:6]: (This isn't a blocker for merging this, still on queue to test.)

Comment (winterheart) [reply dev-python/doit/doit-0.33.1.ebuild:6]: There more and more dependency broken - dev-python/configclass requires dev-python/mergedict

Comment (thesamesam) [reply dev-python/doit/doit-0.33.1.ebuild:6]: Well, someone needs to do it, but I understand that it's not something you have to do either, if you get me. 👍

**Comment by thesamesam** [comment]
Thank you!

**Pull request closed without merging**